packages/stark-ui/src/modules/date-picker/components/date-picker.component.spec.ts

```diff
@@ -114,32 +119,6 @@ describe("DatePickerComponent", () => {
 			component = hostComponent.datePickerComponent;
 		});
 
-		describe("on initialization", () => {
-			it("should set internal component properties", () => {
-				expect(hostFixture).toBeDefined();
-				expect(component).toBeDefined();
-
-				expect(component.logger).not.toBeNull();
-				expect(component.logger).toBeDefined();
-			});
-
-			it("should NOT have any inputs set", () => {
```

Comment:
I think we should keep this unit test at least to be sure what are the initial values of the different properties...